helm/microcosm/templates/osm2pgsql-statefulset.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         {{- if eq .Values.cloudProvider "minikube" }} 
         image: "{{ .Values.osm2pgsql.image.minikube }}"
         {{- end }}
-        command: ['./run.py']
```

Comment:
why do we need to specify the command ?

Reply:
I removed it, but its just if we want to be explicit, or change the command from the default one.